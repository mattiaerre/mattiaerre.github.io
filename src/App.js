import { version } from '../package.json';
import './App.css';

function App() {
  return (
    <main className="App">
      <h1>mattiaerre.github.io</h1>
      <p>Product Engineering at GitHub</p>
      <p>
        <textarea cols="85" rows="20"></textarea>
      </p>
      <footer>{version}</footer>
    </main>
  );
}

export default App;
