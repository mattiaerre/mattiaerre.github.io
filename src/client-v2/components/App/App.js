import React from 'react'; // eslint-disable-line
import EventEmitter from 'wolfy87-eventemitter';
import { version } from '../../../../package.json';
import lockFactory from '../../helpers/lock-factory';
import actionsFactory from '../../helpers/actions-factory';
import Welcome from '../Welcome/Welcome'; // eslint-disable-line
import Actions from '../Actions/Actions'; // eslint-disable-line
import Body from '../Body/Body'; // eslint-disable-line

const eventEmitter = new EventEmitter();

// info: not sure if it is correct to bootstrap the actions here
const lock = lockFactory('YxmJEseG6u9Bl8ONjGkTicUPrryqDVY1', 'mattiaerre.auth0.com', eventEmitter);
const actions = actionsFactory(lock);

const App = () => (
  <div className={`fukol ${version}`}>
    <Welcome nickname="John Doe" eventEmitter={eventEmitter} />
    <Actions actions={actions} eventEmitter={eventEmitter} />
    <Body eventEmitter={eventEmitter} />
  </div>
);

export default App;
