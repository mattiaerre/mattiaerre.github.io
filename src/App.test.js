import { render } from '@testing-library/react';
import App from './App';

test('to match snapshot', () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
