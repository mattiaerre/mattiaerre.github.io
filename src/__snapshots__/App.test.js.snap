// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`to match snapshot 1`] = `
<div>
  <main
    class="App"
  >
    <h1>
      mattiaerre.github.io
    </h1>
    <p>
      Product Engineering at OpenTable
    </p>
    <p>
      <textarea
        cols="100"
        rows="25"
      />
    </p>
    <footer>
      2.1.0
    </footer>
  </main>
</div>
`;
